modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option in order to import the manifest list.
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* You can change the default `importMode` option by running the following command: 
+
++
+[source,terminal]
+----
+oc patch is/cli-artifacts -n openshift -p '{"spec":{"tags":[{"name":"latest","importPolicy":{"importMode":"PreserveOriginal"}}]}}'
+----
```

Comment:
also we want to add that the user can verify that the manifestlist has been imported by inspecting the imagestream tag like so:

`oc get istag cli-artifacts:latest -n openshift -oyaml`

which should list the different manifests inside the manifestlist and the tag reference.

Reply:
Just added a verification step, wdyt? Do you by any chance have an example output of the manifests in the list?

Reply:
this is as example of the tools imagestream tag:
```
[psundara@fedora tmp]$ oc get istag tools:latest -n openshift -oyaml
apiVersion: image.openshift.io/v1
generation: 4
image:
  dockerImageManifestMediaType: application/vnd.docker.distribution.manifest.list.v2+json
  dockerImageManifests:
  - architecture: amd64
    digest: sha256:16d4c96c52923a9968fbfa69425ec703aff711f1db822e4e9788bf5d2bee5d77
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: arm64
    digest: sha256:6ec8ad0d897bcdf727531f7d0b716931728999492709d19d8b09f0d90d57f626
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: ppc64le
    digest: sha256:65949e3a80349cdc42acd8c5b34cde6ebc3241eae8daaeea458498fedb359a6a
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: s390x
    digest: sha256:75f4fa21224b5d5d511bea8f92dfa8e1c00231e5c81ab95e83c3013d245d1719
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  dockerImageMetadata:
    ContainerConfig: {}
    Created: "2022-11-08T22:56:03Z"
    Id: sha256:b3971a8f6def8f1895755692e7e07db4aad5ef8f0f6a78cecc64d1f58f2a33e7
    apiVersion: image.openshift.io/1.0
    kind: DockerImage
  dockerImageMetadataVersion: "1.0"
  dockerImageReference: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b3971a8f6def8f1895755692e7e07db4aad5ef8f0f6a78cecc64d1f58f2a33e7
  metadata:
    creationTimestamp: "2022-11-08T22:56:05Z"
    name: sha256:b3971a8f6def8f1895755692e7e07db4aad5ef8f0f6a78cecc64d1f58f2a33e7
    resourceVersion: "47562"
    uid: 45475624-c094-4d82-908c-b2f77c69aa31
kind: ImageStreamTag
lookupPolicy:
  local: false
metadata:
  creationTimestamp: "2022-11-08T22:56:05Z"
  name: tools:latest
  namespace: openshift
  resourceVersion: "47563"
  uid: e9fc15aa-5339-49e6-afd9-84767fd78662
tag:
  annotations: null
  from:
    kind: DockerImage
    name: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:b3971a8f6def8f1895755692e7e07db4aad5ef8f0f6a78cecc64d1f58f2a33e7
  generation: 4
  importPolicy:
    importMode: PreserveOriginal
    scheduled: true
  name: latest
  referencePolicy:
    type: Source

```

Reply:
Hmm, thats a very large output for an example 😄 . A statement might be just as efficient. Lemme add both and see what we like

Reply:
The large output looked a little jarring in the render so I added a statement instead, wdyt?

Reply:
that makes sense..or add a snippet indicating that the `dockerImageManifests` section will be present when importing manifestlists and add a reference to that:

```
dockerImageManifests:
  - architecture: amd64
    digest: sha256:16d4c96c52923a9968fbfa69425ec703aff711f1db822e4e9788bf5d2bee5d77
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: arm64
    digest: sha256:6ec8ad0d897bcdf727531f7d0b716931728999492709d19d8b09f0d90d57f626
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: ppc64le
    digest: sha256:65949e3a80349cdc42acd8c5b34cde6ebc3241eae8daaeea458498fedb359a6a
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
  - architecture: s390x
    digest: sha256:75f4fa21224b5d5d511bea8f92dfa8e1c00231e5c81ab95e83c3013d245d1719
    manifestSize: 1252
    mediaType: application/vnd.docker.distribution.manifest.v2+json
    os: linux
```

Reply:
I like that, lemme test something! I was thinking that originally and I didn't want to cut out anything important.

Reply:
Yeah I like this new one better